### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by bailanren13** [share121/inter-knot#2443] [其他]
新手绳匠，怎么样才能成为像法厄同那样的传奇绳匠

**Comment by xinyuanyoo** [comment]
开50个空仓库，你就升到了50级，过了绳之道，成为传奇法厄同

**Comment by MrDucker** [comment]
据说新艾丽都有四个超级ai，得到其中一个说不定能让你如有神助<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >
